ecosystem/indexer/migrations/2022-08-08-212440_migrate_token_fields_to_numeric/down.sql

```diff
@@ -0,0 +1,19 @@
+-- This file should undo anything in `up.sql`
+
+ALTER TABLE metadatas
+    ALTER COLUMN seller_fee_basis_points TYPE BIGINT;
+
+ALTER TABLE ownerships
+    ALTER COLUMN amount TYPE BIGINT;
+
+ALTER TABLE token_activities
+    ALTER COLUMN sequence_number TYPE BIGINT;
+
+ALTER TABLE token_datas
```

Comment:
could you please 
* alter the `max_amount` in collections to BIGINT?
* alter the `max_amount` in token_datas to BIGINT as well

Reply:
BIGINT is i64; do we want u64 here?

Reply:
yes, we want to u64